README.md

```diff
@@ -155,6 +188,14 @@ Returns an array of results - one item for each item in `patches`. The type of e
 * `remove`, `replace` and `move` - original object that has been removed
 * `add` (only when adding to an array) - index at which item has been inserted (useful when using `-` alias)
 
+#### jsonpatch.applyPatch (`obj` Object, `patch` Patch, `validate` Boolean) : boolean
+
+Available in *json-patch.js* and *json-patch-duplex.js*
+
+Applies a single `patch` on `obj`.
+
+Returns the modified object.
+
```

Comment:
I am not 100% satisfied with the name either, but note that the type for a single operation is also already called `Patch`, while `apply()` takes an array of that: `Patch[]`. Renaming those would be a breaking change. It also matches how the other libraries name these types.

Operation could be a bit confusing because there is the `op` attribute, which is short for `operation`, that is a string.

In any case I would like to have a short name, `applyOperationObject` is very long.